modos/genomics/region.py

```diff
@@ -104,6 +106,30 @@ def from_ucsc(cls, ucsc: str):
 
         return cls(chrom, start, end)
 
+    @classmethod
+    def from_pysam(
+        cls, record: pysam.VariantRecord | pysam.AlignedSegment
+    ) -> Region:
+        match record:
+            case pysam.VariantRecord:
```

Comment:
```suggestion
            case pysam.VariantRecord():
```
Took me a while to hunt down, but will otherwise throw a type error:
```
TypeError: Argument 'other' has incorrect type (expected pysam.libcalignedsegment.AlignedSegment, got type)
``

Reply:
Ah yes, sorry I missed this one still need to get used to python's match syntax